src/app/dead-letter-queue/dead-letter-queue.component.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {DeadLetterQueueComponent} from './dead-letter-queue.component';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {RageFaceComponent} from '../rage-face/rage-face.component';
+import {DeadLetterQueue} from '../../domain/action-item';
+import {PRIORITY} from '../app.constants';
+
+describe('DeadLetterQueueComponent', () => {
+  let component: DeadLetterQueueComponent;
+  let fixture: ComponentFixture<DeadLetterQueueComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        DeadLetterQueueComponent,
+        RageFaceComponent
+      ]
+    });
+
+    fixture = TestBed.createComponent(DeadLetterQueueComponent);
+    component = fixture.componentInstance;
+  });
+
+  [
+    { call_failed: true, expected_priority: PRIORITY.GREY },
+    { call_failed: false, expected_priority: PRIORITY.RED }
+  ].forEach((inputs) => {
+    it('should create a dead letter queue item appropriately', () => {
```

Comment:
nitpick: might want to include `call_failed` and/or `expected_priority` in the `it` block's description j/ so the test output is distinguishable.

Reply:
ah, good call

Reply:
done